test/src/main/scala/scala/test/twitter_scrooge/BUILD

```diff
@@ -0,0 +1,137 @@
+load("//scala:scala.bzl", "scala_binary", "scala_library")
+load("//twitter_scrooge:twitter_scrooge.bzl", "scrooge_scala_library")
+
+scrooge_scala_library(
+  name = "scrooge1",
+  deps = [
+    "//test/src/main/scala/scala/test/twitter_scrooge/thrift",
+    ":scrooge2_a_srcjar",
+    ":scrooge2_b_srcjar",
+    ":scrooge3_srcjar",
+  ],
+  visibility = ["//visibility:public"],
+)
+
+scrooge_scala_library(
+  name = "scrooge2_a",
+  deps = [
+    "//test/src/main/scala/scala/test/twitter_scrooge/thrift/thrift2:thrift2_a",
+    ":scrooge3_srcjar",
+  ],
+  visibility = ["//visibility:public"],
+)
+
+scrooge_scala_library(
+  name = "scrooge2_b",
+  deps = [
+    "//test/src/main/scala/scala/test/twitter_scrooge/thrift/thrift2:thrift2_b",
+    ":scrooge3_srcjar",
```

Comment:
fixed...this is from the initial runs of it. I did most of my testing with the thinner targets below